docs/reference/configure-fcl.mdx

```diff
@@ -45,21 +90,31 @@ async function addStuff () {
 addStuff().then(d => console.log(d)) // 13 (5 + 7 + 1)
 ```
 
-## Common Configuration Keys
+Examples
+
+
+### Common Configuration Keys
```

Comment:
this is a sub header with the same parent header. this should be changed or remove

Reply:
fixed.